Frontend/lib/data/models/generate_address.dart

```diff
@@ -0,0 +1,20 @@
+class GenerateAddress {
+  GenerateAddress({
+    required this.key1,
+    required this.key2,
+  });
+  late final String key1;
+  late final String key2;
+
+  GenerateAddress.fromJson(Map<String, dynamic> json) {
+    key1 = json['key1'];
```

Comment:
just to be explicit, key1 should be public key 1.

Reply:
Okay well noted